addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -4565,7 +4568,7 @@ public OnClientDisconnect(client)
 	FF2flags[client]=0;
 	Damage[client]=0;
 	uberTarget[client]=-1;
-	playBGM[client]=true;
+	playBGM[client]=false; // Better set it false
```

Comment:
if playBGM[0] is true, then playBGM[client] is true, music for client, but if playBGM[0] is not true, then playBGM[client] is false also, no music for client. OnClientDisconnect resets playBGM[client] to false, with playBGM[0] you can disable music permanently for all clients and new clients